.github/workflows/maven.yml

```diff
@@ -0,0 +1,30 @@
+name: Java CI with Maven
+
+on:
+  push:
+    branches:
+      - '*'
+      - '*/*'
+    paths:
+      - 'src/**'
+      - '**.java'
+  pull_request:
+    branches: [master]
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest, windows-latest]
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up JDK 11
+        uses: actions/setup-java@v1
+        with:
+          java-version: 11
+      - name: Build with Maven
+        run: mvn -B package --file pom.xml
+      - name: Verify maven
+        run: mvn -B clean verify --file pom.xml
```

Comment:
Pourquoi lancer package puis clean puis verify ? (Voir [Introduction to the Build Lifecycle](https://maven.apache.org/guides/introduction/introduction-to-the-lifecycle.html).)

Reply:
Effectivement, le verify englobe le package (on a une redondnace) et peut être enlevé

Reply:
Fix in 819f32f887bf76c4bf4681c63d25a6c132e2b74b